### DISCUSSION TITLE
Species Distribution 1 400 error

### DISCUSSION THREAD

**Discussion by VictoriAragon** [earthlab-education/Earth-Analytics-AY24#559] [Q&A]
Tried the options others have posted about the username and password after getting the 401k error, which fixed that error. But now I am getting this 400 error: 

![image](https://github.com/user-attachments/assets/a46a382f-6f1f-4acb-bdf4-e4268e30b16e)

![image](https://github.com/user-attachments/assets/4b932055-9b1f-4097-a57f-f546315b97de)

Did I do something wrong, or am I missing something?

**Comment by eculler** [comment]
@VictoriAragon I think we solved this in office hours -- looks like it is complaining because the search values are not filled out in lines 8-10
